common/djangoapps/third_party_auth/saml.py

```diff
@@ -293,24 +293,28 @@ def get_user_details(self, attributes):
         try:
             client = self.get_odata_api_client(user_id=username)
             fields = ','.join(self.field_mappings)
+            odata_api_url = '{root_url}User(userId=\'{user_id}\')?$select={fields}'.format(
```

Comment:
Were you planning to log the OData API URL below? If not, you can revert this change.

Reply:
I will revert this.